Tests/Shopware/Connect/Sw41QueryTest.php

```diff
@@ -1,131 +0,0 @@
-<?php
-/**
- * (c) shopware AG <info@shopware.com>
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Tests\ShopwarePlugins\Connect\CategoryQuery;
-
-use Shopware\Connect\Struct\Product;
-use ShopwarePlugins\Connect\Components\ConnectFactory;
-use Tests\ShopwarePlugins\Connect\ConnectTestHelper;
-
-class Sw41QueryTest extends ConnectTestHelper
```

Comment:
It tested a method which was removed in this pr beacuse it was not used in production code